gulp/tasks/banner.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Generate banner on processed source code
+ */
+import gulp from 'gulp';
+import gulpBanner from 'gulp-banner';
+import {isProdBuild} from '../commandLineArgs';
+import {generateBanner, pkgJson, settings} from '../config';
+import merge from 'merge-stream';
+
+const bannerString = '/*! <%= pkgJson.title %> - v<%= pkgJson.version %>\n' +
+	' * <%= pkgJson.author.email %>\n' +
+	' * Copyright ©<%= yearString %> <%= pkgJson.author.name %>\n' +
+	' * <%= dateString %>\n' +
+	' */\n';
+
+function banner(done) {
+	const currentDate = new Date();
+	const yearString = currentDate.getFullYear();
+	const monthString = currentDate.getUTCMonth() > 8 ? currentDate.getUTCMonth() + 1 : `0${currentDate.getUTCMonth() + 1}`;
+	const dayString = currentDate.getDate() > 9 ? currentDate.getDate() : `0${currentDate.getDate()}`;
+	const dateString = `${yearString}-${monthString}-${dayString}`;
```

Comment:
This block could look  less messy using the ECMAScript Internationalization API which should be available in Node 6 and above:

```
const date = new Intl.DateTimeFormat('en-US').format(new Date());
```

or use Date.prototype.toLocaleDateString()

```
const date = new Date().toLocaleDateString('en-US')
```